discretisedfield/tests/test_operators.py

```diff
@@ -15,3 +17,162 @@ def test_integrate():
     assert np.allclose(df.integrate(field * 2), (2000, -4000, 6000))
 
     assert df.integrate(field.plane("z").dot([0, 0, 1])) == 300
+
+
+@pytest.mark.parametrize(
+    "valid",
+    [
+        [True, True, True],
+        [True, True, False],
+        [True, False, True],
+        [False, True, True],
+        [False, False, False],
+        [True, True, False, True],
+        [True, True, False, False, True],
+        [True, False, True, False, True],
```

Comment:
Test a sequence of `False`

```suggestion
        [True, False, True, False, True],
        [False, False, False, True, True, False],
```